doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -205,6 +208,28 @@ def plot_comparison(plot1, plot2, title1, title2):
 
 #####################################################################
 # Let us visualize one restored image, where the dark spots have been
-# inpainted.
+# inpainted. First, we find the contours of the dark spots (i.e., of the mask)
+# so we can draw them on top of the restored image;
+
+contours = measure.find_contours(mask_dilate)
+
+# Gather all (row, column) coordinates of the contours
+x = []
+y = []
+for contour in contours:
+    x.append(contour[:, 0])
+    y.append(contour[:, 1])
+# Flatten them
+x_flat = np.concatenate(x).ravel()
+y_flat = np.concatenate(y).ravel()
+# Create a sparse matrix corresponding to the mask of these contours
+data = np.ones(x_flat.shape, dtype='bool')
+mtx = sparse.coo_matrix((data, (x_flat, y_flat)), shape=mask_dilate.shape)
+# Convert it to array
+arr = mtx.toarray().astype('float')
+
+fig = px.imshow(image_seq_inpainted[50], color_continuous_scale='gray')
+fig.add_trace(go.Contour(z=arr, contours_coloring='lines'))
+plotly.io.show(fig)
 
 plt.show()
```

Comment:
```suggestion

```
clean up on used line of code

Reply:
It is useful in some cases. See comment (and quotes therein) that I previously pointed to: https://github.com/scikit-image/scikit-image/pull/4946#issuecomment-823966459